staging/src/k8s.io/apiserver/pkg/apis/audit/types.go

```diff
@@ -53,6 +77,9 @@ type Event struct {
 	Timestamp metav1.Time
 	// Unique audit ID, generated for each request.
 	AuditID types.UID
+	// Stage of the request handling when this event instance was generated.
+	Stage Stage
```

Comment:
Stage? Phase? Reason? - I'm open to naming suggestions here, but would prefer an enum to a boolean.